controllers/main.go

```diff
@@ -1358,20 +1360,20 @@ func (controller *MainController) Index(c web.C, r *http.Request) (string, int)
 
 // PasswordReset renders the password reset page.
 func (controller *MainController) PasswordReset(c web.C, r *http.Request) (string, int) {
-	t := controller.GetTemplate(c)
 	session := controller.GetSession(c)
 	c.Env["FlashError"] = session.Flashes("passwordresetError")
 	c.Env["FlashSuccess"] = session.Flashes("passwordresetSuccess")
 	c.Env["IsPasswordReset"] = true
-	c.Env["RecaptchaSiteKey"] = controller.recaptchaSiteKey
 	if controller.smtpHost == "" {
 		c.Env["SMTPDisabled"] = true
 	}
+	c.Env["CaptchaID"] = captcha.New()
```

Comment:
No reload button next to the captcha.  To get a new one, all you have to do is reload the page with the protected forms.